src/backends/sqlite3/session.cpp

```diff
@@ -162,10 +180,58 @@ void sqlite3_session_backend::rollback()
     execude_hardcoded(conn_, "ROLLBACK", "Cannot rollback transaction.");
 }
 
+struct SeqCtxt
+{
+    long long value_;
+    bool filledIn_;
+};
+
+static int GetOneLong(void* ctxt, int valueNum, char** values, char**)
+{
+    SeqCtxt* seqCtxt = (SeqCtxt*)ctxt;
+    seqCtxt->value_ = 0;
+    seqCtxt->filledIn_ = true;
```

Comment:
Shouldn't this be only done inside the `if` below? Or do we intentionally do it even if there are no values? If so, could you please add a comment explaining this and why do we do it?